src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -68,6 +70,12 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
   const setInteractiveStateRef = useRef<((state: any) => void)>(setInteractiveState);
   setInteractiveStateRef.current = setInteractiveState;
   const linkedInteractivesRef = useRef(linkedInteractives?.length ? { linkedInteractives } : { linkedInteractives: [] });
+  // const interactiveStateRequestPromise = useRef<Promise<void>>();
```

Comment:
Did you mean to leave these two commented out `const`s in the code?

Reply:
No, fixed, thanks.